### PR TITLE
feat(infra.ci.jenkins.io) add agent VM templates for Linux and Windows as amazon ec2 instances

### PR DESCRIPTION
**Pull request by smerle33**
Closes https://github.com/jenkins-infra/helpdesk/issues/2746

### CODE REVIEW

**Comment by smerle33** [comment]
do not validate for now, work in progress

**Comment by dduportal** [comment]
For info: AWS credentials added for release.ci in https://github.com/jenkins-infra/charts-secrets/commit/9c5bd1a7983692a20ccbad57b134e1419a21432e (private repository)

File: config/ext_jenkins-release.yaml [hunk 1]
```diff
@@ -229,6 +235,125 @@ controller:
...
+                  amiType:
+                    unixData:
+                      sshPort: "22"
+                  associatePublicIp: true
+                  connectBySSHProcess: false
+                  connectionStrategy: PUBLIC_IP
+                  deleteRootOnTermination: true
+                  description: "ARM64 Ubuntu 20.04"
+                  ebsOptimized: false
+                  hostKeyVerificationStrategy: ACCEPT_NEW           # Secure: more flexible to save some time at startup
```
Comment (lemeurherve) [inline config/ext_jenkins-release.yaml:294]:
```suggestion
                  hostKeyVerificationStrategy: ACCEPT_NEW # Secure: more flexible to save some time at startup
```

File: config/ext_jenkins-release.yaml [hunk 2]
```diff
@@ -229,6 +235,125 @@ controller:
...
+                  amiType:
+                    unixData:
+                      sshPort: "22"
+                  associatePublicIp: true
+                  connectBySSHProcess: false
+                  connectionStrategy: PUBLIC_IP
+                  deleteRootOnTermination: true
+                  description: "AMD64 Ubuntu 20.04"
+                  ebsOptimized: false
+                  hostKeyVerificationStrategy: ACCEPT_NEW           # Secure: more flexible to save some time at startup
```
[the remark this answers is not in the document]
Comment (lemeurherve) [reply config/ext_jenkins-release.yaml:258]: Saw your comments block aligned below. Keep it as you put it, or remove all spaces, your choice:)

**Review by dduportal — APPROVED** [review]
good job!

**Pull request merged**